test/model/ModelV2Spec.scala

```diff
@@ -501,46 +501,44 @@ class ModelV2Spec extends WordSpecLike with MustMatchers {
-      originalJourneyConfig.labels.get.en.get.lookupPageLabels.get.manualAddressLinkText must contain(resolvedJourneyConfig.labels.en.lookupPageLabels.manualAddressLinkText)
-
-      originalJourneyConfig.labels.get.en.get.editPageLabels.get.title must contain(resolvedJourneyConfig.labels.en.editPageLabels.title)
-      originalJourneyConfig.labels.get.en.get.editPageLabels.get.heading must contain(resolvedJourneyConfig.labels.en.editPageLabels.heading)
-      originalJourneyConfig.labels.get.en.get.editPageLabels.get.line1Label must contain(resolvedJourneyConfig.labels.en.editPageLabels.line1Label)
-      originalJourneyConfig.labels.get.en.get.editPageLabels.get.line2Label must contain(resolvedJourneyConfig.labels.en.editPageLabels.line2Label)
-      originalJourneyConfig.labels.get.en.get.editPageLabels.get.line3Label must contain(resolvedJourneyConfig.labels.en.editPageLabels.line3Label)
-      originalJourneyConfig.labels.get.en.get.editPageLabels.get.townLabel must contain(resolvedJourneyConfig.labels.en.editPageLabels.townLabel)
-      originalJourneyConfig.labels.get.en.get.editPageLabels.get.postcodeLabel must contain(resolvedJourneyConfig.labels.en.editPageLabels.postcodeLabel)
-      originalJourneyConfig.labels.get.en.get.editPageLabels.get.countryLabel must contain(resolvedJourneyConfig.labels.en.editPageLabels.countryLabel)
-      originalJourneyConfig.labels.get.en.get.editPageLabels.get.submitLabel must contain(resolvedJourneyConfig.labels.en.editPageLabels.submitLabel)
-
-      originalJourneyConfig.labels.get.en.get.confirmPageLabels.get.title must contain(resolvedJourneyConfig.labels.en.confirmPageLabels.title)
-      originalJourneyConfig.labels.get.en.get.confirmPageLabels.get.heading must contain(resolvedJourneyConfig.labels.en.confirmPageLabels.heading)
-      originalJourneyConfig.labels.get.en.get.confirmPageLabels.get.infoSubheading must contain(resolvedJourneyConfig.labels.en.confirmPageLabels.infoSubheading)
-      originalJourneyConfig.labels.get.en.get.confirmPageLabels.get.infoMessage must contain(resolvedJourneyConfig.labels.en.confirmPageLabels.infoMessage)
-      originalJourneyConfig.labels.get.en.get.confirmPageLabels.get.submitLabel must contain(resolvedJourneyConfig.labels.en.confirmPageLabels.submitLabel)
-      originalJourneyConfig.labels.get.en.get.confirmPageLabels.get.searchAgainLinkText must contain(resolvedJourneyConfig.labels.en.confirmPageLabels.searchAgainLinkText)
-      originalJourneyConfig.labels.get.en.get.confirmPageLabels.get.changeLinkText must contain(resolvedJourneyConfig.labels.en.confirmPageLabels.changeLinkText)
-      originalJourneyConfig.labels.get.en.get.confirmPageLabels.get.confirmChangeText must contain(resolvedJourneyConfig.labels.en.confirmPageLabels.confirmChangeText)
-
-      resolvedJourneyConfig.labels.cy mustBe None
+      originalJourneyConfig.labels.get.en.get.appLevelLabels.get.navTitle mustBe resolvedJourneyConfig.labels(true).appLevelLabels.navTitle
```

Comment:
these tests don't need to be changed as the added boolean has no effect on the code